src/app/public/modules/text-editor/services/text-editor-adapter.service.ts

```diff
@@ -100,6 +100,27 @@ export class SkyTextEditorAdapterService {
     }
   }
 
+  public toggleEditorAbility(id: string, disabled: boolean, focusableChildren: HTMLElement[]): void {
```

Comment:
I'm concerned the word `ability` in this method name doesn't provide a clear indication of what this method does. What would you say to following Angular's form control patterns and creating `.disable()` and `.enable()` methods? They can each call the same logic so you don't need to replicate code. The idea here is that this should be more familiar to devs working with forms.

Reply:
fixed in commit: ["clarify functionality"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/6bd3f94b822cc3465b72b294ad6d322e83595e61)